server/routes/public.js

```diff
@@ -29,15 +29,48 @@ router.get('/api/login', async (req, res) => {
 });
 
 router.get('/api/create-user', async (req, res) => {
+    // joi verification
+    // Validate appropriate parameters are passed into the create account endpoint.
+    const paramSchema = Joi.object({
+       
+       //The following joi verifications make sure the values inputted adhere to the format requirements
+       //specified in the migrations> create-worker-table and migrations> create-student-table.
+
+        email: Joi.string().email({ minDomainSegments: 2, tlds: { allow: ['com', 'net', 'ca'] } }).max(320).required(),
+        password: Joi.string().min(3).max(40).required(),
+        phone: Joi.string().max(20).phoneNumber(),
+        firstName: Joi.string().min(1).max(20).required(),
+        lastName: Joi.string().min(1).max(20).required(),
+        studentID: Joi.number().positive().integer(),
```

Comment:
Hi Nancy,
Again for consistency with the remainder of the backend and frontend code, I would recommend adjust `studentID` to `studentId` and `workerID` to `workerId`.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've changed the name from workerID to workerId. I have also made similar changes to studentId.

You can now see these changes in the following locations:
line 44 and 45 of server/routes/public.js
line 15 of server/models/db>worker.js
line 18 of server/models/db>worker.js
line 19 of server/models/db>worker.js
line 14 of server/models/db>student.js
line 17 of server/models/db>student.js
line 18 of server/models/db>student.js